README.rst

```diff
@@ -177,33 +177,19 @@ Look at the new Repository Version created
         "number": 1
     }
 
-Create a ``docker`` Publisher ``baz``
-----------------------------------------------
-
-``$ http POST http://localhost:8000/pulp/api/v3/publishers/docker/docker/ name=baz``
-
-.. code:: json
-
-    {
-        "_href": "/pulp/api/v3/publishers/docker/8ce1b34c-56c3-4ced-81b8-81e83b174fbc/",
-        ...
-    }
-
-``$ export PUBLISHER_HREF=$(http :8000/pulp/api/v3/publishers/docker/docker/ | jq -r '.results[] | select(.name == "baz") | ._href')``
-
 
-Use the ``bar`` Publisher to create a Publication
+Create a Publication
```

Comment:
Given the new endpoint name, I think we should call this "publish a repository version"